exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -392,28 +391,27 @@ func addSingleDoubleHistogramDataPoint(pt *otlp.DoubleHistogramDataPoint, metric
 	countlabels := createLabelSet(pt.GetLabels(), nameStr, baseName+countStr)
 	addSample(tsMap, count, countlabels, metric)
 
-	// count for +Inf bound
-	var totalCount uint64
+	// cumulative count for conversion to cumulative histogram
+	var cumulativeCount uint64
 
-	// process each bound, ignore extra bucket values
+	// process each bound, based on histograms proto definition, # of buckets = # of explicit bounds + 1
 	for index, bound := range pt.GetExplicitBounds() {
 		if index >= len(pt.GetBucketCounts()) {
 			break
 		}
-		bk := pt.GetBucketCounts()[index]
+		cumulativeCount += pt.GetBucketCounts()[index]
 		bucket := &prompb.Sample{
-			Value:     float64(bk),
+			Value:     float64(cumulativeCount),
 			Timestamp: time,
 		}
 		boundStr := strconv.FormatFloat(bound, 'f', -1, 64)
 		labels := createLabelSet(pt.GetLabels(), nameStr, baseName+bucketStr, leStr, boundStr)
```

Comment:
This can be optimized to avoid re-calculating the labels from the pt.GetLables().